### PR TITLE
DEV-3805 add new rust exercise

### PR DESCRIPTION
**Pull request by nprimo**

### CODE REVIEW

**Review by mikysett — CHANGES_REQUESTED** [review]
Maybe let's clarify what a lucas number is to allow more focus on the recursion part

File: subjects/lucas_number/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Complete the body of the **function** `lucas_number`.
+
+```rust
+pub fn lucas_number(n: u32) -> u32 {}
+```
+
+This function receives a number `n` and returns the `n`th number in the Lucas Numbers. 
+
+The Lucas Numbers start like this: 2, 1, 3, 4, 7, 11, 18, 29, 47, 76, 123, etc... 
+
```
[the remark this answers is not in the document]
Comment (nprimo) [reply subjects/lucas_number/README.md:14]: Agree! I will add this

**Review by mikysett — APPROVED** [review]
LGTM

**Review by mikysett — APPROVED** [review]
LGTM

**Pull request merged**